modules/installing-ove-iso.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-ove.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-iso_{context}"]
+= Downloading the installation ISO
+
+You must first download the ISO image that will be used to run the installation on your bare-metal machines.
+// Not sure if the following sentence is technically accurate or even a necessary detail to mention at this point in the process.
+This image includes all of the necessary {product-title} release images, which will be "pinned" to the cluster, to eliminate the need for an image registry when installing in a disconnected environment.
```

Comment:
I would avoid the term "pinned" here, as it vaguely reminds a feature already in GA (the PinnedImageFeatureSet), which is not involved in this case. It's ok to cite that the ISO contains all the necessary release payload images, and a basic set of OLM operators required to install the Virtualization pane. It could be useful to list them explicitly.
In addition, I'd add a big disclaimer about the fact that ISO size is not standard (currently ~36GB), as this could have some relevant impacts (ie download times/hw support)

Reply:
That's my mistake, I thought pinned image sets were an aspect of the appliance builder that was used to avoid the need for a registry. Either way, that might have been a technical detail that didn't have to be mentioned.

I'll reword to just focus on the images/operator files it includes and add a note about the file size not being standard.